RAG.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
In indexing job, you can directly call `CollectionService.ingest_data({collection_name:})`